TodoListService/Models/ClaimInfo.cs

```diff
@@ -7,8 +7,6 @@ namespace TodoListService.Models
 {
     public class ClaimInfo
     {
-        public ClaimInfo() { }
```

Comment:
Indeed, if there are no constructors defined, the compiler always emits a default (empty) one. So it's common to not specify any for simple "data" classes such as this one.